scripts/get_das_info.py

```diff
@@ -88,15 +90,17 @@ def convert_for_top_queries(data: dict) -> str:
         ),
     ),
 )"""
-    elif dataset_type == "xxx":
+    elif dataset_type == "comment":
         # comment out this dataset
         return f"""        # {dataset_type}=DatasetInfo(
         #     keys=[
-        #         "{data['name']}",  # noqa
+        #         "xxx",  # noqa
```

Comment:
Just noticed it after the changes, but why is `{data['name']}` replaced by `xxx` here? Probably it doesn't make sense to have a commented-out block with out the actual DAS key.

Reply:
but maybe it's better to keep the key of the identifier_map there (e.g. "_MT-175p5_"

Reply:
Ah ok, having `comment` as the `DatasetInfo` name is probably not that informative, so using the `identifier_map` key instead makes sense.